packages/core/src/App/Containers/RealityCheckModal/reality-check-modal.jsx

```diff
@@ -0,0 +1,138 @@
+import { Field } from 'formik';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { withRouter } from 'react-router';
+import { Input } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { getPreBuildDVRs, validNumber } from 'Utils/Validator/declarative-validation-rules';
+import BriefModal from './brief-modal.jsx';
+import SummaryModal from './summary-modal.jsx';
+
+const IntervalField = ({ values, touched, errors, handleChange, handleBlur }) => (
+    <div className='reality-check__fieldset'>
+        <Field name='interval'>
+            {({ field }) => (
+                <Input
+                    {...field}
+                    data-lpignore='true'
+                    type='text'
+                    label={localize('Time interval')}
+                    value={values.interval}
+                    onChange={handleChange}
+                    onBlur={handleBlur}
+                    hint={localize('Interval should be between 10-60 minutes')}
+                    required
+                    error={touched.interval && errors.interval}
+                    autoComplete='off'
+                    maxLength='2'
+                />
+            )}
+        </Field>
+    </div>
+);
+
+const RealityCheckModal = withRouter(
```

Comment:
since it's a functional component it's possible to use the `useHistory` hook from react-router (instead of the withRouter HOC)